test/schema-examples.test.js

```diff
@@ -509,6 +509,60 @@ test('should return custom error messages with ajv-errors', t => {
   })
 })
 
+test('should be able to handle formats of ajv-formats when added by plugins option', t => {
+  t.plan(3)
+
+  const fastify = Fastify({
+    ajv: {
+      plugins: [
+        require('ajv-formats')
```

Comment:
It is actually added by `@fastify/ajv-compiler`.

The only exception is when you apply your own `ajv-formats`, it will override the `ajv-formats` in `@fastify/ajv-compiler`

Reply:
I had no idea `ajv-formats` was included with Fastify via `@fastify/ajv-compiler` — that's great! I don't see it mentioned in the [Validation](https://www.fastify.io/docs/latest/Reference/Validation-and-Serialization/) documentation, but I'd be happy to open a PR that adds this in.

As far as I can tell, `ajv-formats` wasn't included with Fastify v3 — is that correct?

Reply:
> As far as I can tell, ajv-formats wasn't included with Fastify v3 — is that correct?

No, `ajv-formats` is embedded inside `ajv@6`.
It is extracted in `ajv@7` and we need to explicitly add it through `ajv-formats`

Reply:
Oh that's right, I forgot about that. And Fastify went from Ajv v6 (formats embedded) -> v8 (formats in ajv-formats)?